ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -542,6 +542,7 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
           id = "INVALID_DEDUPLICATION_PERIOD",
           ErrorCategory.InvalidGivenCurrentSystemStateOther,
         ) {
+      val ValidMaxDeduplicationFieldKey = "max_deduplication_duration"
```

Comment:
I don't see a rename here.

Reply:
Don't open PRs at 17:30 before the weekend 💀 
fixed